### PR TITLE
LRQA-82691

### PR DESCRIPTION
**Pull request by Claudioss23**

### CODE REVIEW

**Comment by Claudioss23** [comment]
ci:test:sf

**Comment by Claudioss23** [comment]
ci:test:sf

**Comment by Claudioss23** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+				mimeType = "text/plain",
+				sourceFileName = ${dmEntryFile1});
+
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = ${dmEntryTitle2},
+				groupName = "Guest",
+				mimeType = "application/msword",
+				sourceFileName = ${dmEntryFile2});
+
+			Refresh();
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase:66]: Hey, can we try add for loops here too? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+				dmDocumentTitle = ${dmEntryTitle1});
+
+			Click(locator1 = "Icon#BACK");
+
+			DMDocument.viewPG(
+				dmDocumentFileName = ${dmEntryFile2},
+				dmDocumentTitle = ${dmEntryTitle2});
+
+			Click(locator1 = "Icon#BACK");
+		}
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase:79]: You can also add here :slightly_smiling_face:

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase (hunk 1 above)
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase:66]: Okay, will do. Thank you for the review!

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase (hunk 2 above)
Comment (Claudioss23) [reply portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase:79]: Okay. Thanks!

**Comment by Claudioss23** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]
LGTM. Thanks for the test and the updates! :muscle:

**Pull request closed without merging**